### ISSUE TITLE
code issues

### ISSUE THREAD

**Issue by Liuhao-128** [JuliousHurtado/meta-training-setup#1]
Hi, Hurtado, I am interesting in your work. 
I have a questions about the code in approach.py.  Specifically, I am not understand what is the meaning of input_feats=batch[2] .
The shape of input_feats is [batchsize, channel, 28, 28].
Could your tell me the answer, thanks a lot.

**Comment by JuliousHurtado** [comment]
Hi, 
thank you for being interested in our work.

About your question. 
If you check the function \_\_getitem\_\_ in the dataloaders of the different benchmarks, you can observed that it has 3 output.

- batch[0]: is the input of the model (images)
- batch[1]: is the corresponding label
- batch[2]: is the feature vector of a pre-trained model

This last output is nor really used in the basic method proposed, only in the experiment when we use a pre-trained model as an F function.
